modules/eks/variables.tf

```diff
@@ -67,6 +67,7 @@ variable "network_info" {
         az_id     = string
       }))
     })
+    vpc_cidrs = string
```

Comment:
are we missing `eips` here?

Reply:
I didn't add it to reduce impact, but I can't include it if you want